src/olympia/wsgi.py

```diff
@@ -10,27 +6,11 @@
 from django.core.wsgi import get_wsgi_application
 
 
-log = logging.getLogger('z.startup')
-
-# Remember when mod_wsgi loaded this file so we can track it in nagios.
-wsgi_loaded = datetime.now()
-
-
 # Do validate and activate translations before running the app.
 django.setup()
 django.utils.translation.activate(django.conf.settings.LANGUAGE_CODE)
 
-# This is what mod_wsgi runs.
-django_app = get_wsgi_application()
-
 
-# Normally we could let WSGIHandler run directly, but while we're dark
-# launching, we want to force the script name to be empty so we don't create
-# any /z links through reverse.  This fixes bug 554576.
+# This is referenced in docker/uwsgi.ini: module = olympia.wsgi:application
 def application(env, start_response):
-    if 'HTTP_X_ZEUS_DL_PT' in env:
-        env['SCRIPT_URL'] = env['SCRIPT_NAME'] = ''
-    env['wsgi.loaded'] = wsgi_loaded
-    env['hostname'] = django.conf.settings.HOSTNAME
-    env['datetime'] = str(datetime.now())
-    return django_app(env, start_response)
+    return get_wsgi_application()(env, start_response)
```

Comment:
I'm guessing/hoping there was no reason why `django_app` was declared outside of the function?

Reply:
mmm, just in case I'm going to replace that with `application = get_wsgi_application()` to be closer to django's default implementation.

Reply:
https://github.com/mozilla/addons-server/pull/18621/commits/f5814670990ce54609b41be74363cac740d52dfe